.aws/deploy/backend-task-definition.prod.json

```diff
@@ -0,0 +1,221 @@
+{
+  "containerDefinitions": [
+    {
+      "name": "backend",
+      "portMappings": [
+        {
+          "containerPort": 8081,
+          "hostPort": 8081,
+          "protocol": "tcp"
+        }
+      ],
+      "essential": true,
+      "environment": [],
+      "mountPoints": [
+        {
+          "sourceVolume": "efs",
+          "containerPath": "/efs",
```

Comment:
Sanity check here: I remember that in our previous code the path used to be `repos/efs`, just confirming here that there is no typo here ya

Reply:
from the oncall run book, it seems to be `/efs`

<img width="405" alt="Screenshot 2023-12-21 at 10 04 10 AM" src="https://github.com/isomerpages/isomercms-backend/assets/44049504/ae566e23-fdb6-43f8-b811-eabc18843205">

Reply:
https://www.notion.so/opengov/On-Call-Runbook-ab4bffa2472d47ef9b657242632da7f9?pvs=4#b0faa42dee264794bd7a01d2e73b4243
can send the link to the place where you got this? let me go and update

Reply:
oh wait sorry can ignore since we mount on /efs, then get the repos + repos-lite path from there